### ISSUE TITLE
[NBug] Значение не может быть неопределенным.Parameter name: value

### ISSUE THREAD

**Issue by yadimetrio** [gitextensions/gitextensions#9471]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Error Details
```
System.ArgumentNullException: Значение не может быть неопределенным.
Имя параметра: value
   в ConEmu.WinForms.ConEmuStartInfo.set_ConsoleProcessCommandLine(String value)
   в GitUI.CommandsDialogs.FormBrowse.<FillTerminalTab>b__201_0(Object sender, TabControlCancelEventArgs args)
   в System.Windows.Forms.TabControl.OnSelecting(TabControlCancelEventArgs e)
   в System.Windows.Forms.TabControl.WmSelChange()
   в System.Windows.Forms.TabControl.WndProc(Message& m)
   в GitUI.CommandsDialogs.FullBleedTabControl.WndProc(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   в System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   в System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

## Additional information
click on "Console" window

## Environment
- Git Extensions 3.5.2.12435
- Build 77413699a9c25b160a6a622311529372b5d87f5b
- Git 2.29.2.windows.2 (recommended: 2.30.0 or later)
- Microsoft Windows NT 10.0.19042.0
- .NET Framework 4.8.4390.0
- DPI 96dpi (no scaling)

**Comment by gerhardol** [comment]
Please set the console in settings.
![image](https://user-images.githubusercontent.com/6248932/128926300-f9fe582e-88a3-4af0-a139-0d4db5e469f8.png)

**Comment by gerhardol** [comment]
Also fixed in `release/3.5`
